samples/template-operator/README.md

```diff
@@ -37,19 +37,19 @@ Additionally, it hides Kubernetes boilerplate code to develop fast and efficient
 
 ### Generate kubebuilder operator 
 
-1. Create a folder and make a directory, e.g. `operator`.
+1. Create a folder and make a directory, e.g. `test-operator`.
```

Comment:
@adityabhatia, do you mean this `operator` is the operator folder under root folder? like `lifecycle-manager/operator`? Then I think the first sentence is misleading, to me, it' means the root folder name. 
```suggestion
1. Create module folder, e.g. `test-operator` and make a directory `operator`
```

Reply:
Yes that's right. Check [Makefile](https://github.com/kyma-project/lifecycle-manager/blob/259a8216545a7d7aec8dd8d6ce5440db497da55f/samples/template-operator/Makefile) 
and look for `operator/..`

_"Create `module` folder"_ is confusing. I would suggest _"In your root directory"_ to be more generic.